crates/bevy_math/src/common_traits.rs

```diff
@@ -0,0 +1,177 @@
+use glam::{Quat, Vec2, Vec3, Vec3A, Vec4};
+use std::fmt::Debug;
+use std::ops::{Add, Div, Mul, Sub};
+
+/// A type that supports the mathematical operations of a vector space, irrespective of dimension.
```

Comment:
```suggestion
/// A type that supports the mathematical operations of a real vector space, irrespective of dimension.
```
Nit: this is explicitly not a complex vector space, so we might as well clarify it here.